0101-nix formatting.md

```diff
@@ -0,0 +1,65 @@
+---
+feature: nix_formatting
+start-date: 2021-08-17
+author: Raphael Megzari
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: Timothy DeHerrera (nrdxp), 0x4A6F, Silvan Mosberger (infinisil)
```

Comment:
```suggestion
shepherd-team: Timothy DeHerrera (nrdxp), 0x4A6F, Silvan Mosberger (infinisil), piegames
```
I'm assuming that piegames has agreed to shepherd, based on their participation in the meeting and that this was one of the outcomes listed in the notes :)